Dockerfile.dapper

```diff
@@ -1,27 +1,18 @@
-FROM ubuntu:20.04
+FROM registry.suse.com/suse/sle15:15.3
 
 ARG DAPPER_HOST_ARCH=amd64
 ARG http_proxy
 ARG https_proxy
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
 
-ENV DEBIAN_FRONTEND=noninteractive
-
-RUN apt-get update && \
-    apt-get dist-upgrade -y && \
-    apt-get install -y gcc ca-certificates git wget curl vim less file tox python-dev iptables libdevmapper-dev libltdl7 && \
-    rm -f /bin/sh && ln -s /bin/bash /bin/sh
-
-ENV DOCKER_URL_amd64=https://download.docker.com/linux/ubuntu/dists/focal/pool/stable/amd64/docker-ce-cli_20.10.7~3-0~ubuntu-focal_amd64.deb \
-    DOCKER_URL_arm64=https://download.docker.com/linux/ubuntu/dists/focal/pool/stable/arm64/docker-ce-cli_20.10.7~3-0~ubuntu-focal_arm64.deb \
-    DOCKER_URL=DOCKER_URL_${ARCH}
-
-RUN wget ${!DOCKER_URL} -O docker_ce_${ARCH} && dpkg -i docker_ce_${ARCH}
+RUN zypper -n install gcc ca-certificates git wget curl vim less file python3-tox python3-devel iptables libdevmapper1_03 libltdl7 awk && \
+    rm -f /bin/sh && ln -s /bin/bash /bin/sh && \
+    zypper install -y -f docker
 
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
-RUN wget -O - https://storage.googleapis.com/golang/go1.14.1.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
+RUN wget -O - https://storage.googleapis.com/golang/go1.13.15.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
```

Comment:
According to go.mod file, longhorn-manager is using golang v1.13

Reply:
Don’t think we need to revert back to 1.13. The go version mentioned in go.mod should be min version.

Reply:
@PhanLe1010 since we updated kubernetes min version, did we end up having to update the min go version as well?
To be in line with the kubernetes/grpc/ etc versions :)

Reply:
We didn't update the min go version since Kubernetes v1.18 is still using go 1.13

Reply:
will use the latest version of golang 1.13 which is v1.13.15, and we can discuss the upgrade separately later.